<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDistanceBinarysDlg</class>
 <widget class="QDialog" name="QDistanceBinarysDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>526</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>开入、开出量设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QTabWidget" name="m_tabIn_Out">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabInput">
      <attribute name="title">
       <string>开入量</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2" colspan="3">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">border:0px;

</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="m_radio_or">
            <property name="text">
             <string>逻辑或</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="m_radio_and">
            <property name="text">
             <string>逻辑与</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QCheckBox" name="m_checkD">
         <property name="text">
          <string>开入D</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="m_checkB">
         <property name="text">
          <string>开入B</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="m_checkC">
         <property name="text">
          <string>开入C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="m_checkA">
         <property name="text">
          <string>开入A</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_checkE">
         <property name="text">
          <string>开入E</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QCheckBox" name="m_checkH">
         <property name="text">
          <string>开入H</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="m_checkG">
         <property name="text">
          <string>开入G</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="m_checkF">
         <property name="text">
          <string>开入F</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_tabOutput">
      <attribute name="title">
       <string>开出量</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QLabel" name="m_lab_B04">
         <property name="text">
          <string>开出4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_lab_B01">
         <property name="text">
          <string>开出1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="m_cmb_B01">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="m_labDelayTime">
         <property name="text">
          <string>开出保持时间(s)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="m_lab_B05">
         <property name="text">
          <string>开出5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="m_lab_B03">
         <property name="text">
          <string>开出3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="m_cmb_B05"/>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="m_cmb_B04"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_lab_B02">
         <property name="text">
          <string>开出2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="m_cmb_B02"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="m_lab_B06">
         <property name="text">
          <string>开出6</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="m_cmb_B07"/>
       </item>
       <item row="7" column="2">
        <widget class="QComboBox" name="m_cmb_B08"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="m_lab_B08">
         <property name="text">
          <string>开出8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="m_cmb_B03"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="m_lab_SwitchMode">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>开出翻转启动方式</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="m_cmb_B06"/>
       </item>
       <item row="9" column="2">
        <widget class="QSettingLineEdit" name="m_txtDelayTime"/>
       </item>
       <item row="8" column="2">
        <widget class="QComboBox" name="m_cmb_SwitchMode"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="m_lab_B07">
         <property name="text">
          <string>开出7</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="m_labKeepTime">
         <property name="text">
          <string>开出翻转时刻(s)</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QSettingLineEdit" name="m_txtKeepTime">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnOK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSettingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../UI/Controls/SettingCtrls/qsettinglineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
